ocean-client/README.md

```diff
@@ -126,6 +142,7 @@ Enabled: keeps your address clean by using commissions (dust) to payback loans a
 Disabled: will not touch commissions (dust), only what is needed by default calculations
 
 ### VAULTMAXI_SWAP_REWARDS_TO_MAIN
+DEPRECATED since the new reinvest-pattern
```

Comment:
maybe we should warn those people who have this activated. that they need to adapt to the new reinvest pattern on update. i mean yes, they will see the new parameter. but we need to describe this IMHO

Reply:
no need to do anything. if no reinvest pattern is active, this variable still counts. 
![image](https://user-images.githubusercontent.com/29012906/200913063-229cef6c-c116-43bd-9e7b-75ddc291433f.png)